src/askcode/lib/AskCode.ts

```diff
@@ -30,8 +30,8 @@ export class AskCode {
 
   parent?: AskCode;
 
-  toString(): string {
-    return askCodeToSource(this);
+  toString(pretty: boolean = false): string {
```

Comment:
let's take an option object here instead, so:

```ts
toString({
  pretty = false,
}: { pretty?: boolean } = {}): string {
```

This why it will be easier to develop this public API

Reply:
Right